packages/instant/src/containers/selected_asset_buy_order_state_button.tsx

```diff
@@ -1,20 +1,78 @@
+import { AssetBuyer, BuyQuote } from '@0x/asset-buyer';
```

Comment:
I think this should be a `.ts` file because we don't have any tsx in the file, this is the same for any other connected containers that we might write

Reply:
Good point, fixed in 68182fb6c41256f9886c7aa0269746b3a9ac632f